docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -674,6 +674,35 @@ Default: `256MB`
 
 ---
 
+### Network compression
+
+Use the following two gflags to configure RPC compression:
+
+##### --enable_stream_compression
+
+Controls whether YugabyteDB uses RPC compression. If you set this flag to `true`, you must also specify a stream compression algorithm.
```

Comment:
This part was probably unclearly described. You do not *have to* also specify the algo. But the other way around, you cannot specify an algo, until you have enabled compression support.